hoot-services/src/main/java/hoot/services/controllers/grail/WaitOverpassUpdate.java

```diff
@@ -73,7 +73,7 @@ public CommandResult execute() {
         commandResultStart.setJobId(jobId);
         commandResultStart.setStdout("Starting wait on overpass sync...\n");
         commandResultStart.setStderr("");
-        commandResultStart.setPercentProgress(0);
+        commandResultStart.setPercentProgress(100);
```

Comment:
Are these changes because we want to show the entire job as completed and just waiting on overpass sync, rather than having the overpass sync be the last portion of the job completed?

Reply:
yea it just made sense to make this 100% since the command of "starting wait" isnt a real command so its instant

Reply:
I think I'm catching on, the start and finish commands are just so something is in the job logs, the java services code in between actually does the polling.

Reply:
yup that is correct